### DISCUSSION TITLE
ScanResult - Device Charging

### DISCUSSION THREAD

**Discussion by ffeliciodeveloper** [Navideck/universal_ble#44] [General]
Good morning/Good afternoon/Good evening everyone.

Firstly, congratulations on the initiative to create the library.

I would like to ask for help to find out if in the ScanResult method, there is a way to indicate that there are no more devices to be loaded?

In the tests I carried out, I saw that ScanResult does not have any indication to inform that the scanning operation was completed.

Is there a specific way within the library to know if the scan has finished?

Thank you all!

**Comment by rohitsangwan01** [comment]
@ffeliciodeveloper there is no way in the library to detect if there are no more scanResults to load, but you can apply tricks to detect
, like a timer which will check if no new device discovered in given time
